qiskit/visualization/pulse/matplotlib.py

```diff
@@ -323,9 +324,10 @@ def draw(self, pulse: Waveform,
         # the suptitle line, however since the font style can take on a type of None
         # we need to unfortunately check both the type and the value of the object.
         if isinstance(self.style.title_font_size, int) and self.style.title_font_size > 0:
-            figure.suptitle(str(pulse.name),
-                            fontsize=self.style.title_font_size,
-                            y=bbox.y1 + 0.02,
+            title_label = str('')
+            if draw_title:
+                title_label = str(pulse.name)
+            figure.suptitle(title_label, fontsize=self.style.title_font_size, y=bbox.y1 + 0.02,
                             va='bottom')
```

Comment:
Won't adding an empty title cause there to be wasted space in the plot canvas?

Reply:
We need to pass a title so old version was adding auto-generated one, I think empty/default name is more suitable

Reply:
Could you only add the `suptitle` if `draw_title` is `True`?